proto/regen/ecocredit/v1/query.proto

```diff
@@ -186,6 +186,11 @@ service Query {
   rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {
     option (google.api.http).get = "/regen/ecocredit/v1/params";
   }
+
+  // CreditType queries credit type information by abbreviation.
+  rpc CreditType(QueryCreditTypeRequest) returns (QueryCreditTypeResponse) {
+    option (google.api.http).get = "/regen/ecocredit/v1/credit-types/{abbreviation}";
```

Comment:
We previously updated the endpoints so that we provide one that matches the query name (same as sdk modules) and an alternative that uses the more conventional format. Would be nice to be consistent here.

```suggestion
    option (google.api.http) = {
      get : "/regen/ecocredit/v1/credit-type/{abbreviation}"
      additional_bindings : [ {get : "/regen/ecocredit/v1/credit-types/{abbreviation}"} ]
    };
```